include/tvm/runtime/container/optional.h

```diff
@@ -93,6 +93,11 @@ class Optional : public ObjectRef {
     ICHECK(data_ != nullptr);
     return T(data_);
   }
+  /*!
+   * \return The internal object pointer with container type of T.
+   * \note This function do not perform not-null checking.
+   */
+  const ContainerType* get() const { return static_cast<ContainerType*>(data_.get()); }
```

Comment:
Note that get is not a standard API as per stl style for Optional. While `get` is a standard stl style name. We might want to have a longer name in this case.

How about `GetContainerPtr`(use CamelCase to indicate that it is not STL compatible).

The comment can be updated to explain the behavior of nullopt case(return nullptr) and non-nullopt case(returns the container ptr)

Reply:
Thanks @tqchen for the valuable feedback! On the other hand, I would say this PR actually does is to simply improve the `Optional::get` method to return the underlying type instead of the plain Object pointer, and hence it might make sense to keep the name as it is. What do you think>?